test/e2e/egress_test.go

```diff
@@ -664,6 +666,89 @@ func testEgressMigration(t *testing.T, data *TestData, triggerFunc, revertFunc f
 	checkIPNeighbor(fromNode)
 }
 
+func testEgressUpdateBandwidth(t *testing.T, data *TestData) {
+	skipIfEgressShapingDisabled(t)
+	skipIfNotIPv4Cluster(t)
+	skipIfHasWindowsNodes(t)
+	var (
```

Comment:
no need got a `var` block, you can just use `:=` assignment for each one

Reply:
Changed.